src/via-structures.stanza

```diff
@@ -549,10 +553,28 @@ defmethod make-via-structure (v:DifferentialViaStructure -- pose:Pose = DEF_VS_L
 
     port COMMON
 
-    net via-net-P (sig-in.P, sig-out.P)
-    topology-segment(sig-in.P, sig-out.P)
-    net via-net-N (sig-in.N, sig-out.N)
-    topology-segment(sig-in.N, sig-out.N)
+    val [via-def-P, via-def-N] = match(via-def(v)):
+      (obj1:Via): [obj1, obj1]
+      (obj2:[Via, Via]): obj2
+    val via-pitch = pitch(v) / 2.0
+
+    ; First wrapper instance
+    inst via-pin-P : topo-via-pin(`p)
```

Comment:
The only concern I have here is the symbol for the diff-pair via structures being two separate schematic symbols in this approach. I think that will make the schematic harder to read. Assuming the user uses the `create-via-structure` function - then at least these two symbols will be in the same sch group so they will be group together a bit. 
I think this is fine for now until we can integrate via structures more directly in the JITX runtime.